Modules/Devices/src/Devices.Application/IntegrationEvents/Outgoing/IdentityDeletionProcessStartedIntegrationEvent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus.Events;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+
+namespace Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
+
+public class IdentityDeletionProcessStartedIntegrationEvent : IntegrationEvent
+{
+    public IdentityDeletionProcessStartedIntegrationEvent(Identity identity, IdentityDeletionProcess deletionProcess) : base($"{identity.Address}/DeletionProcessStarted")
```

Comment:
The deletion process id should be part of the event id. Otherwise the duplicate detection of the event buses may filter consecutive events out.